src/js/common/components/form-elements/ErrorableTextInput.jsx

```diff
@@ -96,7 +96,7 @@ class ErrorableTextInput extends React.Component {
             name={this.props.name}
             tabIndex={this.props.tabIndex}
             autoComplete={this.props.autocomplete}
-            type="text"
+            type={this.props.type}
```

Comment:
This seemed like the most reasonable change, rather than creating something like `ErrorableEmailInput` and `ErrorableTelInput`, which would result in some mostly duplicated components. We could rename ErrorableTextInput, but it didn't seem worth the widespread change it'd be.